inductiva/tasks/streams.py

```diff
@@ -65,23 +107,161 @@ def __on_message(self, unused_ws, message):
         streams = data.get("streams")
         for stream in streams:
             msg = stream["values"][0][1]
-            print(msg, file=self.fout)
+            self._write_message(msg)
+
+    def _write_message(self, msg):
+        if self._fout_isatty:
+            # 1. move up SPACING lines
+            # 2. move to beginning of line and clear line
+            # 3. write message
+            # 4. add new line
+            # 5. move down SPACING lines
+            # 6. move to beginning of line and clear line
+            # 7. add new blank line
+            s = f'{UP}\r{CLEAR_LINE}{msg}\n{DOWN}\r{CLEAR_LINE}\n'
+        else:
+            s = msg + '\n'
```

Comment:
But we should set `s=msg` first.

Reply:
they are not the same. Anyway, I changed the implementation in [e457844](https://github.com/inductiva/inductiva/pull/1037/commits/e4578449a77fafc1bbe2ed083af5f20ed5c3b1c5) so that this doesn't apply anymore.